### PR TITLE
PYIC-2214: Update journey engine state machine config for new check existing identity journey.

### PR DESCRIPTION
**Pull request by Danomanic**
### What changed

Update journey engine state machine config for new check existing identity journey.

### Issue tracking

- [PYIC-2214](https://govukverify.atlassian.net/browse/PYIC-2214)

### CODE REVIEW

File: lambdas/process-journey-step/src/main/resources/statemachine/build/ipv-core-main-journey.yaml [hunk 1]
```diff
@@ -58,6 +58,17 @@ MITIGATION_JOURNEY:
 INITIAL_IPV_JOURNEY:
   name: INIT
   parent: null
+  events:
+    next:
+      type: basic
+      name: next
+      targetState: CHECK_EXISTING_IDENTITY
```
Comment (dannyhayes-gds) [inline lambdas/process-journey-step/src/main/resources/statemachine/build/ipv-core-main-journey.yaml:76]: I think this event is being applied to the wrong state. Currently this means we are checking the for an existing identity as the very first thing in IPV Core. But from the figma journey diagram, I think we first need to display the IPV_IDENTITY_START_PAGE then do the check after that

Comment (kerrr) [reply lambdas/process-journey-step/src/main/resources/statemachine/build/ipv-core-main-journey.yaml:76]: Yeah, sorry, I think my description was wrong in the ticket was wrong.

Comment (kerrr) [reply lambdas/process-journey-step/src/main/resources/statemachine/build/ipv-core-main-journey.yaml:76]: I've updated the ticket with the correct description (I think).

Comment (Danomanic) [reply lambdas/process-journey-step/src/main/resources/statemachine/build/ipv-core-main-journey.yaml:76]: Corrected this!

**Review by dannyhayes-gds — APPROVED** [review]

**Pull request merged**